mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/ProfileFragment.java

```diff
@@ -135,6 +138,12 @@ public void onUserImageEditClicked() {
     public void onEditProfileClicked() {
         if (getActivity() != null) {
             getActivity().startActivity(new Intent(getActivity(), EditProfileActivity.class));
```

Comment:
There's no need for this line as later you are already creating an intent `PassCodeActivity` the very next line